File: 프로그래머스/2/42584. 주식가격/주식가격.py
def solution(prices):
    N = len(prices)
    
    result = []
    
    for i in range(N-1):
        cnt = 0
        for j in range(i, N-1):
            if prices[i] <= prices[j]:
                cnt += 1
            else:
                break
        result.append(cnt)
    result.append(0)
            
    return result

File: 프로그래머스/2/42584. 주식가격/주식가격.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution(vector<int> prices) {
    int N = prices.size();
    
    vector<int> result;
    for (int i = 0; i < N-1; ++i)
    {
        int cnt = 0;
        for (int j = i; j < N-1; ++j)
        {
            if (prices[i] <= prices[j])
            {
                cnt++;
            }
            else
            {
                break;
            }
        }
        result.push_back(cnt);
    }
    result.push_back(0);
    
    return result;
}